aggregator/endpoints/query.py

```diff
@@ -40,6 +40,37 @@ async def send_beacon_query(request):
 
     return results
 
+async def post_beacon_query(request):
+    """Post Beacon queries and respond synchronously."""
+    LOG.info("Normal response (sync), inside .............post_beacon_query")
+
+    jsonObj = request.headers["query"]
+    LOG.info("json passed:......:"+str(jsonObj))
+
+    # Task variables
+    params = request.query_string  # query parameters (variant search)
+    
+
+    tasks = []  # requests to be done
+    services = await get_services(request.host)  # service urls (beacons, aggregators) to be queried
+    access_token = await get_access_token(request)  # Get access token if one exists
+
+    for service in services:
+        # Generate task queue
+        LOG.info("now calling service.........:"+str(service)+":params:"+str(params)+":access_token:"+str(access_token)+":request json:"+str(request.headers["query"]))

```

Comment:
use f strings, also this should be debug as printing access token in log is not advisable, unless code is in development or testing
```suggestion
        LOG.debug(f"now calling service.........:{service}:params:{params}:access_token: {access_token}:request json:{request.headers['query']})

```